modin/pandas/test/test_series.py

```diff
@@ -3391,51 +3391,35 @@ def test_update(data, other_data):
 @pytest.mark.parametrize("normalize, bins, dropna", [(True, 3, False)])
 def test_value_counts(normalize, bins, dropna):
     # We sort indices for Modin and pandas result because of issue #1650
-    modin_series, pandas_series = create_test_series(test_data_values[0])
-
-    modin_result = sort_index_for_equal_values(
-        modin_series.value_counts(normalize=normalize, ascending=False), False
-    )
-    pandas_result = sort_index_for_equal_values(
-        pandas_series.value_counts(normalize=normalize, ascending=False), False
-    )
-    df_equals(modin_result, pandas_result)
-
-    modin_result = sort_index_for_equal_values(
-        modin_series.value_counts(bins=bins, ascending=False), False
-    )
-    pandas_result = sort_index_for_equal_values(
-        pandas_series.value_counts(bins=bins, ascending=False), False
+    eval_general(
+        *create_test_series(test_data_values[0]),
+        lambda df: df.value_counts(normalize=normalize, ascending=False),
+        comparator=df_equals_with_non_stable_indices,
```

Comment:
the same test cases but rewritten using `eval_general` to make it simpler to read